.github/workflows/upload-benchmarks.yml

```diff
@@ -0,0 +1,75 @@
+name: Upload-benchmarks
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - '**.md'
+  push:
+    branches:
+      - develop
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build-core:
+    name: Build semgrep-core
+    runs-on: ubuntu-latest
+    container: returntocorp/ocaml:alpine
+    steps:
+      - name: Pre-checkout fixes
+        run: |
+          sudo chmod -R 777 /github
```

Comment:
what is this? Do we do that in the other workflow?

Reply:
Yes, this is copied from tests.yml, the build and test semgrep-core job.

Reply:
put a comment then, like coupling: this is copy-pasted from tests.yml#build and test semgrep-core job